spring-cloud-stream-binder-kinesis-docs/src/main/asciidoc/overview.adoc

```diff
@@ -458,15 +458,15 @@ awslocal kinesis create-stream --stream-name my-test-stream --shard-count 1
 
 awslocal dynamodb create-table \
 --table-name spring-stream-lock-registry \
---attribute-definitions AttributeName=lockKey,AttributeType=S AttributeName=sortKey,AttributeType=S \
---key-schema AttributeName=lockKey,KeyType=HASH AttributeName=sortKey,KeyType=RANGE \
+--attribute-definitions AttributeName=lockKey,AttributeType=S \
+--key-schema AttributeName=lockKey,KeyType=HASH \
```

Comment:
This is correct fix since starting with version `3.0` `spring-integration-aws` provides its own table definition via `DynamoDbLockRepository`